.gitignore

```diff
@@ -34,3 +34,15 @@ yarn-error.log*
 
 # vercel
 .vercel
+
+# PWA files
```

Comment:
I took the example from 
https://blog.usman-s.me/how-to-make-a-nextjs-app-a-pwa-with-offline-support